src/Plugin/InitGraphQlTranslations.php

```diff
@@ -0,0 +1,93 @@
+<?php
+/**
+ * ScandiPWA_CmsGraphQl
```

Comment:
personally, I'd put it in PersistedQuery module rather than CmsGraphQl, because it already has plugins on the GrapQL controller